packages/e2e-test-utils/src/tab-through-block.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	externalWrapperHasFocus,
+	inserterToggleHasFocus,
+	tabThroughBlockMoverControl,
+	tabThroughBlockToolbar,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Tabs through a content block and asserts that the external wrapper, inserter toggle, mover controls, and toolbar buttons all receive keyboard focus.
+ *
+ * @param {string} blockType  The expected value of the data-type attribute of the block's external wrapper
+ * @returns {Promise} A promise that's resolved when the browser has finished tabbing through the major components of a common block.
+ */
+
+export async function tabThroughBlock( blockType ) {
```

Comment:
Shouldn't the name be more explicit? Something like:

```suggestion
export async function tabThroughBlockControls( blockType ) {
```

If we opt for such name then we probably should update also `tabThroughBlockMoverControl()` to be `tabThroughBlockMovers()`.